src/harness/runner.ts

```diff
@@ -207,6 +207,11 @@ function beginTests() {
         ts.Debug.enableDebugInfo();
     }
 
+    // run tests in en-US by default.
+    const savedUILocale = ts.getUILocale();

```

Comment:
I think you need to `savedUILocale` inside the call to `beforeEach` in case the locale was set sometime between the declaration of `savedUILocale` and the test running.

Reply:
Highly unlikely, but I'll make the change.